global_install_scripts/install.bash

```diff
@@ -104,6 +104,7 @@ function run() {
 	function set_talisman_binary_name() {
 		# based on OS (linux/darwin) and ARCH(32/64 bit)
 		declare SUFFIX
+		declare OS
```

Comment:
Hey @tinamthomas , good catch, I made a mistake by not using an IDE, this error could've been avoided. Could you please change 127  to SUFFIX instead of OS and get rid of the OS variable declaration ? That would be more inline with the changes I've made.